packages/react/__tests__/exports.ts

```diff
@@ -12,18 +9,8 @@ import * as internal from '../dist/internal';
 describe('@aws-amplify/ui-react', () => {
   describe('exports', () => {
     it('should match snapshot', () => {
-      expect(Object.keys(exported)).toMatchInlineSnapshot(`
+      expect(Object.keys(exported).sort()).toMatchInlineSnapshot(`
```

Comment:
Can't comment on the lines outside of this PR but just calling out that we would need to add this to the other snapshot tests below as well!

Reply:
Can we create "sorted" variables for the values used in the `expect` lines to make them a bit cleaner?
```
const sortedExports = Object.keys(exported).sort();
```

Reply:
Also, since we are cleaning up this file already, what do you think of just using `toMatchSnapshot` in place of `toMatchInlineSnapshot` so we can remove the string arguments?

Reply:
I added all of the above suggestions.